asset/js/setup.js

```diff
@@ -33,10 +33,15 @@ function setupAnchors() {
   });
 }
 
+function fadeOutSpinner() {
+  jQuery('#spinner-overlay').fadeOut(100);
```

Comment:
As stated earlier, this should be `.remove()` rather than fade out, to make it more pleasant for the user to navigate.